docs/learning-the-basics.md

```diff
@@ -11,7 +11,16 @@ At the top of the screen you can see the Panel. On the left is the Applications
 ![The Panel](../images/docs/learning-the-basics/panel.png)
 
 #### Applications {#applications}
-On the left side of the panel is the Applications item. Clicking **Applications** brings up a launcher with all of your installed apps. You can view multiple pages of apps using the pagers at the bottom or by scrolling. You can also use the view switcher at the top to switch between a grid view and a category view. Lastly, you can search the launcher by typing at any time.
+On the left side of the panel is the Applications item. Clicking **Applications** brings up a launcher with all of your installed apps. You can use the view switcher at the top to switch between a grid view and a category view. You can view multiple pages of apps using the pagers at the bottom or by scrolling.
```

Comment:
I think we should switch these two sentences around considering you can only use the pager while on grid view, which is default

```
You can view multiple pages of apps using the pagers at the bottom or by scrolling. You can also use the view switcher at the top to switch between a grid view and a category view.
```